manifests/d/DynalistInc/Dynalist/1.0.6/DynalistInc.Dynalist.locale.en-US.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: DynalistInc.Dynalist
+PackageVersion: 1.0.6
+PackageLocale: en-US
+Publisher: Dynalist Inc.
+PublisherUrl: https://dynalist.io/
+PublisherSupportUrl: https://help.dynalist.io/
+PrivacyUrl: https://dynalist.io/privacy
+PackageName: Dynalist
+PackageUrl: https://dynalist.io/download
+License: ©Dynalist, all rights reserved.

```

```suggestion
License: Proprietary
```
A copyright isn't a license; I believe they have their own EULA, which would make this a Proprietary license